zerver/openapi/python_examples.py

```diff
@@ -689,9 +689,10 @@ def get_streams(client: Client) -> None:
 
 
 @openapi_test_function("/streams/{stream_id}:patch")
-def update_stream(client: Client, stream_id: int) -> None:
+def update_stream(client: Client) -> None:
     # {code_example|start}
-    # Update the stream by a given ID
+    # Update channel with a given channel ID
+    stream_id = client.get_stream_id("python-test")["stream_id"]
```

Comment:
Note that the section inside the comments here is part of the documentation -- I don't think our current pattern is to put a hardcoded weird stream name like "python-test" in that documentation. 

The motive for passing these parameters BTW was just to increase robustness by having us, say, update/delete an object we'd created earlier in the test sequence; I'm not sure I understand the motive for changing that detail. Though see https://github.com/zulip/zulip/pull/30115 for an attempt to make these more robust.

Reply:
I was feeling the tension between the variable name being passed to the function (`stream_id`) and the descriptive language being "channel" as we're in this transition period. But that's possibly not really an issue.

Reply:
Updated to not show getting the `stream_id` value in the user-facing documentation and removed any other changes that would have done the same.